keps/sig-node/3695-pod-resources-for-dra/README.md

```diff
@@ -274,8 +274,9 @@ These cases will be added in the existing e2e tests:
 
 #### Beta
```

Comment:
were there any template updates since it was in alpha? I would think it was written long ago, and I expect more changes in the tamplate.